aten/src/ATen/native/transformers/attention.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <c10/util/Optional.h>
+
+namespace at {
+class Tensor;
+namespace native {
+std::tuple<Tensor, Tensor, Tensor> transform_bias_rescale_qkv_cuda(
+    const Tensor& qkv,
+    const Tensor& qkv_bias,
+    const int64_t num_head);
+void masked_softmax_dropout_cuda(
+    const Tensor& attn_scores,
+    const c10::optional<Tensor>& attn_mask);
```

Comment:
I think you shouldn't need to explicitly define the header declarations here; you can get them from `NativeFunctions.h` or `ATen/ops/_transform_bias_rescale_qkv.h` (need to case on `AT_PER_OPERATOR_HEADERS`)

Reply:
will try removing

Reply:
looks like we can just delete the header and replace it with nothing.